preprocessing/nextclade/src/loculus_preprocessing/prepro.py

```diff
@@ -139,7 +241,7 @@ def enrich_with_nextclade(  # noqa: C901, PLR0912, PLR0914, PLR0915
     unaligned_nucleotide_sequences: dict[
         AccessionVersion, dict[SegmentName, NucleotideSequence | None]
     ] = {}
-    error_dict: dict[AccessionVersion, list[ProcessingAnnotation]] = {}
+    alerts: Alerts = Alerts(errors={}, warnings={})
```

Comment:
We should just initialize alerts with empty lists then we save a lot of the `.get(id,[])` stuff

Reply:
done